gatsby/content/blog/2022/03/2022-03-22-synapse-1.55.mdx

```diff
@@ -0,0 +1,72 @@
+---
+date: '2022-03-22'
+title: Synapse 1.55 released
+categories:
+- Releases
+author: Brendan Abolivier
+---
+
+Hi all, [Synapse
+1.55](https://github.com/matrix-org/synapse/releases/tag/v1.55.0) is out! Let's
+see the main talking points of this new release.
+
+## Removing support for Mjolnir 1.3.1 and older
+
+Administrators of large homeservers and communities will already be familiar
+with [Mjolnir](https://github.com/matrix-org/mjolnir), which is a tool designed
+to help them moderate a large number of rooms in an efficient way. It includes a
+bot as well as a Synapse module to better interface with the homeserver.
+
+Due to a change in how we manage some of Synapse's internal utilities, this
+release of Synapse breaks compatibility with Mjolnir versions 1.3.1 and older.
+Homeserver administrators using Mjolnir and upgrading to Synapse 1.55 should
+make sure they're running Mjolnir version 1.3.2 or later.
+
+See the [upgrade
+notes](https://matrix-org.github.io/synapse/v1.55/upgrade#compatibility-dropped-for-mjolnir-131-and-earlier)
+for more information.
+
+## Moving `synctl`
+
+`synctl` is a tool provided as part of Synapse to run and manage your instance
+and its workers (if any).
+
+As part of our work to enable reproducible builds in Synapse, we have recently
```

Comment:
Could you briefly explain how this helps with reproducible builds?

Reply:
Have reworded this to make it clearer poetry, which will then enable reproducible builds, is the thing that needs it.